alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -56,10 +57,16 @@ Proceed with the following steps to fix the issue:
 1. Use the logs to guide you to the place where the conflicting targets are defined.
 2. Remove the duplicated targets or ensure that distinct targets are labeled uniquely.
 
-### The logs only revolve around the same scrape pool
+### Target is exposing duplicated samples for the same timestamp
+
+In this scenario, even if the samples have the same value for the same
+timestamp, they are considered duplicates.
+
+#### NOTE
 
-This might mean that the target exposes duplicated samples. In this scenario, even
-if the samples have the same value, they are considered duplicates.
+A regression introduced in OpenShift Container Platform 4.16.0 might cause the alert
+to fire when a target is exposing samples of the same series with different explicit
+timestamps. ([OCPBUGS-39179](https://issues.redhat.com/browse/OCPBUGS-39179))
```

Comment:
Just to be sure, this case will go away once https://github.com/openshift/prometheus/pull/224 merges? So eventually the runbook should say "Only affects OCP 4.16.0 to 4.16.xyz".

Reply:
Yep, but I cannot decide on the patch version now, that's why I linked the ticket so the fix can be tracked.

Reply:
(suggestion)
```suggestion
timestamps (check [OCPBUGS-39179](https://issues.redhat.com/browse/OCPBUGS-39179) to track the resolution).
```